ssh/key

```diff
@@ -0,0 +1,39 @@
+-----BEGIN OPENSSH PRIVATE KEY-----
```

Comment:
Was this committed in error?

Reply:
It's a (hopefully) dummy SSH key generated by @eranco74, I've taken it from the BiP SNO PoC branch in the installer repo

Reply:
No worries, It's not my private key.

Reply:
I didn't check it into this repo for a reason - there should never be a need to check a private SSH key into git. Even if you think it's safe, it's almost sure to cause lots of people in futue to wonder whether it's a leak

Reply:
https://github.com/eranco74/bootstrap-in-place-poc/pull/4